src/components/DuosHeader.css

```diff
@@ -104,6 +104,19 @@
   transition: background 0.3s ease !important;
 }
 
+.contact-us-button {
+  color: #ffffff;
+  font-size: 16px;
+  border: none;
+  background-color: transparent;
+  outline: none;
+  transition: background 0.3s ease !important;
+}
+
+.contact-us-button:hover {
+  color: #5BB0CA !important;
+}
```

Comment:
In general, I try to avoid as much css as possible in separate css files. Instead, I like to use inline styling. The hover case, however, is more complicated to implement inline. There are some small examples in the codebase of inlined hoverstate, but I'm fine with this.